<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changeWidget</class>
 <widget class="QDialog" name="changeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1188</width>
    <height>815</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1188</width>
    <height>815</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1188</width>
    <height>815</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="imageButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>321</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="imageButton1">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>321</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="imageButton1_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>300</y>
     <width>321</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="imageButton1_3">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>300</y>
     <width>321</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="imageButton1_4">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>50</y>
     <width>321</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="imageButton1_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>321</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="imageButton1_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>560</y>
     <width>321</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="imageButton1_7">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>560</y>
     <width>321</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="imageButton1_8">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>560</y>
     <width>321</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
